### ISSUE TITLE
The "Close" option is written in German

### ISSUE THREAD

**Issue by anirbandey303** [MyLibreLab/MyLibreLab#29]
When you click on File option inside the application, the option to "Close" the application is written in German as "schließen".
Given that rest of the application's options are in English, I think this should be translated to English too.
![Screenshot](https://user-images.githubusercontent.com/29774651/94863638-613db800-0458-11eb-855b-c8a7ad99efd3.png)

**Comment by marko-radosavljevic** [comment]
I completely agree. We are aware of this problem, but thanks for opening an issue! ^^

Every GUI element text and its tooltip should be in English. This should be priority over other translation work.
